### ISSUE TITLE
faceit proof ?

### ISSUE THREAD

**Issue by useking** [iseekwonderful/csgoGlow#276]
just a question, is this cheat faceit proof ?

**Comment by wholivesinapineappleunderthesea** [comment]
No, you can't play Faceit on Mac because the AC was never released for Mac.

Even if it was, you can't write to cs memory... The best we can do on Faceit is reading memory and an overlay for box/bone esp...
